src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -924,7 +924,7 @@ export default class EthereumApi implements Api {
       }
 
       const block = new RuntimeBlock(
-        Quantity.from((parentHeader.number.toBigInt() || 0n) + 1n),
+        parentHeader.number.add(1n),
```

Comment:
Since we `add(1n)` more than anything else I wonder if a highly optimized `.increment()` function would be worth while. Maybe we do all the Math in a Buffer directly or something. :thinking: (this probably can/should be saved for another PR)

Reply:
I'll look into that. Initially I did all of the math directly in the buffer, and got somewhat promising results, but I considered the complexity to be unreasonable for the minimal gains. It is likely much more significant for a simple increment() function however, as the conversion to bigint and back is a more singificant percentage of the entire operation. I will look into it.